src/multi_vector_simulator/cli.py

```diff
@@ -166,16 +171,16 @@ def main(**kwargs):
 
     print("")
     logging.debug("Accessing script: D0_modelling_and_optimization")
-    results_meta, results_main = modelling.run_oemof(
+    results_meta, results_main = D0.run_oemof(
         dict_values, save_energy_system_graph=save_energy_system_graph,
     )
 
     print("")
     logging.debug("Accessing script: E0_evaluation")
-    evaluation.evaluate_dict(dict_values, results_main, results_meta)
+    E0.evaluate_dict(dict_values, results_main, results_meta)
 
     logging.debug("Accessing script: F0_outputs")
-    output_processing.evaluate_dict(
+    F0.evaluate_dict(
```

Comment:
below this you don't wan to also use `F0.store_as_json` for `json_with_results.json`as well ?

Reply:
I am not sure for what this would be - the json processed json results are already stored somewhere, and this would only add the log-messages (I think) to the json results. I dont aim to fix the #765 with this, so I think its fine.